sql/moz-fx-data-shared-prod/mozilla_vpn_derived/waitlist_funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH send_ids AS (
+  SELECT
+    SendID,
+    ARRAY_AGG(DISTINCT URL) AS URLs,
+  FROM
+    `mozilla-cdp-prod`.sfmc.clicks
+  WHERE
+    URL LIKE "%://vpn.mozilla.org%"
+  GROUP BY
+    SendID
+),
+jobs AS (
+  SELECT
+    SendID,
+    SentTime,
+    URLs,
+  FROM
+    `mozilla-cdp-prod`.sfmc.sendjobs
+  INNER JOIN
+    send_ids
+  USING
+    (SendID)
+)
+SELECT
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.sent
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS sent,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.bounces
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS bounced,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.opens
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS opened,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.unsubs
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS unsubscribed,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.clicks
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+      AND URL IN UNNEST(URLs)
+  ) AS clicked,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.clicks
+    JOIN
+      jobs
+    USING
+      (SendID)
+    JOIN
+      `moz-fx-data-shared-prod`.mozilla_vpn_external.users_v1
```

Comment:
I tried joining on `waitlist_v1`, and then filtering on `user_id IS NOT NULL`, because that was supposed to be an indicator that a waitlist member was converted, but that resulted in 0 matches :/